CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 ## master / unreleased
 
-## 0.13.0 / 2022-11-15
-
+## 0.13.0 / 2023-01-25
+
+* [BUGFIX] Fixes suspected duplicate label panic for some GCP metric [#153](https://github.com/prometheus-community/stackdriver_exporter/pull/153)
```

Comment:
We usually just put the `#PRNUM` rather than put the full link. GitHub expands the PR URL for us at render time.

Also, please sort in the order of `CHANGE`, `FEATURE`, `ENHANCEMENT`, `BUGFIX`.